eng/pipelines/optprof.yml

```diff
@@ -8,8 +8,11 @@ resources:
     source: NuGet.Client-Official
     trigger:
       branches:
+        include:
         - dev
         - release-*
+        exclude:
+        - release-6.0.x
```

Comment:
similar changes were made in release-6.3.x branch already https://github.com/NuGet/NuGet.Client/commit/66d48f468a14e0e1246f73deff808442e686eff5